lightpath/happi/containers.py

```diff
@@ -0,0 +1,32 @@
+"""
+Define subclasses of Happi Items for use with Lightpath
+"""
+import re
+from copy import deepcopy
+
+from happi.item import EntryInfo, OphydItem
+
+
+class LightpathItem(OphydItem):
```

Comment:
I take it this intentionally does not inherit from `LCLSItem` to make it accessible to those not at the LCLS?

Reply:
Yep!  Though the only test cases I have are based off the LCLS facility.

Reply:
Should be :+1: to eventually inherit this in pcdsdevices

Reply:
The conflict shows itself as:
``` 
(pcds-5.4.1)roberttk@psbuild-rhel7-01:~/devrepos/pcdsdevices(enh_lp_mixin +)$ happi container-registry
File "/cds/home/r/roberttk/devrepos/happi/happi/containers.py", line 208, in load
    self._safe_add(entry_name, var)
  File "/cds/home/r/roberttk/devrepos/happi/happi/containers.py", line 173, in _safe_add
    raise RuntimeError(f"Duplicated entry found. Keys: {key} "
RuntimeError: Duplicated entry found. Keys: pcdsdevices.happi.containers.LightpathItem and lightpath.happi.containers.LightpathItem point to same class: <class 'lightpath.happi.containers.LightpathItem'>
```

When you import `from lightpath.happi.containers import LightpathItem` in `pcdsdevices.happi.containers`

It seems the container gets collected once in lightpath and again in pcdsdevices under different keys, since they're being collected from different entrypoints.  I can add an issue for it

Reply:
I see now - yeah, this is a problem. Needs more thought